### PR TITLE
README updated

### PR DESCRIPTION
**Pull request by AkhilKamath**
Small change in the README file.
(alternate commands for windows)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,17 @@ If you are new to github, follow [git-task](https://github.com/OSDLabs/git-task)
...
+$ cd src
+$ pip install -r requirements.txt
+```
+
+* For windows use:
+
+```
+$ virtualenv waves
+$ waves\Scripts\activate
+$ git clone https://github.com/OSDLabs/WavesWebsite2016 src
```
Comment (SebastinSanty) [inline README.md]: No need of a separate code block for windows, just mention, "use `waves\Scripts\activate` instead of `source bin/activate` " as a NOTE.

**Comment by SebastinSanty** [comment]
I have forgot to add pyc files to gitignore (my bad). But can you remove other files staged for commit which aren't necessary (like db.sqlite and .pyc files). Use `git add README.md` instead of `git add .`

**Comment by AkhilKamath** [comment]
ok

**Comment by SebastinSanty** [comment]
Can you please squash both the commits into a single commit?

Edit: three commits into one

**Comment by AkhilKamath** [comment]
done....just check

**Comment by SebastinSanty** [comment]
No its not done. I can still see 3 commits. Use `git rebase -i HEAD~3` to squash them.

**Comment by AkhilKamath** [comment]
yes i did that....i'll check again

**Comment by SebastinSanty** [comment]
Another Pull Request was submitted and was merged.

**Pull request closed without merging**